agent/internal/client/methods.go

```diff
@@ -865,3 +865,30 @@ func (c *communicatorImpl) GetDockerLogs(ctx context.Context, hostID string, sta
 
 	return body, nil
 }
+
+func (c *communicatorImpl) ConcludeMerge(ctx context.Context, patchId, project, status string, td TaskData) error {
+	info := requestInfo{
+		method:   http.MethodGet,
+		path:     fmt.Sprintf("commit_queue/%s/conclude_merge", patchId),
+		version:  apiVersion2,
+		taskData: &td,
+	}
+	body := struct {
```

Comment:
If this is standard for this file then I guess it's fine, but I would preference defining an actual struct called ConcludeMergeInput or something, that has Project and Status, and passing that into concludeMerge, rather than having three strings all in a row.

Reply:
the next ticket removes one of the parameters so it's only 2 strings, which makes things a bit better. If it gets unwieldy I'll change it but I think it's fine for now